_layouts/default.html

```diff
@@ -11,7 +11,7 @@
 		<meta name="publisher" content="simplabs" />
     <title>{{page.title}}</title>
     <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css">
-    <link href="//fonts.googleapis.com/css?family=Source+Code+Pro:300,400,500" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet">
```

Comment:
no, this was taken directly from the Google Fonts website itself, they no longer suggest the use of protocol relative URLs.